### ISSUE TITLE
ANR Input dispatching timed out

### ISSUE THREAD

**Issue by ssoher** [playgameservices/play-games-plugin-for-unity#2601]
```

"main" tid=1 Blocked
"main" prio=5 tid=1 Blocked
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x725a69f0 self=0xac835000
  | sysTid=7789 nice=-10 cgrp=default sched=0/0 handle=0xb09da4a4
  | state=S schedstat=( 6449106078 3717965707 10525 ) utm=428 stm=215 core=5 HZ=100
  | stack=0xbe25c000-0xbe25e000 stackSize=8MB
  | held mutexes=
 
  at com.google.android.gms.ads.internal.webview.v.E (com.google.android.gms.policy_ads_fdr_dynamite@20290004@20290004.249178941.249178941)
- waiting to lock <0x0fe119da> (a com.google.android.gms.ads.internal.webview.v) held by thread 5
 
  at com.google.android.gms.ads.internal.webview.t.E (com.google.android.gms.policy_ads_fdr_dynamite@20290004@20290004.249178941.249178941:8)
 
  at com.google.android.gms.ads.internal.webview.j.onPageFinished (com.google.android.gms.policy_ads_fdr_dynamite@20290004@20290004.249178941.249178941:2)
- locked <0x01549e0b> (a java.lang.Object)
 
  at com.android.webview.chromium.Ap.onPageFinished (WebViewContentsClientAdapter.java:221)
 
  at org.chromium.android_webview.AwContentsClientCallbackHelper$MyHandler.handleMessage (AwContentsClientCallbackHelper.java:70)
 
  at android.os.Handler.dispatchMessage (Handler.java:106)
 
  at android.os.Looper.loop (Looper.java:164)
 
  at android.app.ActivityThread.main (ActivityThread.java:6626)
 
  at java.lang.reflect.Method.invoke (Native method)
 
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:438)
 
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:811)
"FinalizerDaemon" tid=5 Blocked
"FinalizerDaemon" daemon prio=5 tid=5 Blocked
  | group="system" sCount=1 dsCount=0 flags=1 obj=0x14a00338 self=0xac836e00
  | sysTid=7797 nice=4 cgrp=default sched=0/0 handle=0x94de4970
  | state=S schedstat=( 29476039 40938387 85 ) utm=1 stm=0 core=4 HZ=100
  | stack=0x94ce2000-0x94ce4000 stackSize=1038KB
  | held mutexes=
 
  at com.google.android.gms.ads.internal.webview.j.k (com.google.android.gms.policy_ads_fdr_dynamite@20290004@20290004.249178941.249178941:3)
- waiting to lock <0x01549e0b> (a java.lang.Object) held by thread 1
 
  at com.google.android.gms.ads.internal.webview.v.destroy (com.google.android.gms.policy_ads_fdr_dynamite@20290004@20290004.249178941.249178941:4)
- locked <0x0fe119da> (a com.google.android.gms.ads.internal.webview.v)
 
  at com.google.android.gms.ads.internal.webview.t.destroy (com.google.android.gms.policy_ads_fdr_dynamite@20290004@20290004.249178941.249178941:10)
 
  at com.google.android.gms.ads.nonagon.ad.interstitial.b.finalize (com.google.android.gms.policy_ads_fdr_dynamite@20290004@20290004.249178941.249178941:2)
 
  at java.lang.Daemons$FinalizerDaemon.doFinalize (Daemons.java:250)
 
  at java.lang.Daemons$FinalizerDaemon.runInternal (Daemons.java:237)
 
  at java.lang.Daemons$Daemon.run (Daemons.java:103)
 
  at java.lang.Thread.run (Thread.java:764)
"Jit thread pool worker thread 0" tid=2 Native
"Jit thread pool worker thread 0" daemon prio=5 tid=2 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14a001a0 self=0xac835c00
  | sysTid=7794 nice=9 cgrp=default sched=0/0 handle=0xa5497970
  | state=S schedstat=( 1327849893 1214837248 1629 ) utm=109 stm=22 core=4 HZ=100
  | stack=0xa5399000-0xa539b000 stackSize=1022KB
  | held mutexes=
 
  #00  pc 0000000000018d68  /system/lib/libc.so (syscall+28)
 
  #01  pc 00000000000b77d1  /system/lib/libart.so (art::ConditionVariable::WaitHoldingLocks(art::Thread*)+80)
 
  #02  pc 000000000038ff31  /system/lib/libart.so (art::ThreadPool::GetTask(art::Thread*)+172)
 
  #03  pc 000000000038f8a7  /system/lib/libart.so (art::ThreadPoolWorker::Run()+62)
 
  #04  pc 000000000038f4c5  /system/lib/libart.so (art::ThreadPoolWorker::Callback(void*)+92)
 
  #05  pc 00000000000475af  /system/lib/libc.so (__pthread_start(void*)+22)
 
  #06  pc 000000000001afc9  /system/lib/libc.so (__start_thread+32)
"ReferenceQueueDaemon" tid=4 Waiting
"ReferenceQueueDaemon" daemon prio=5 tid=4 Waiting
  | group="system" sCount=1 dsCount=0 flags=1 obj=0x14a002b0 self=0xac836800
  | sysTid=7796 nice=4 cgrp=default sched=0/0 handle=0x94ee9970
  | state=S schedstat=( 12521927 14458285 57 ) utm=0 stm=0 core=6 HZ=100
  | stack=0x94de7000-0x94de9000 stackSize=1038KB
  | held mutexes=
 
  at java.lang.Object.wait (Native method)
- waiting on <0x00b0d4e8> (a java.lang.Class<java.lang.ref.ReferenceQueue>)
 
  at java.lang.Daemons$ReferenceQueueDaemon.runInternal (Daemons.java:178)
- locked <0x00b0d4e8> (a java.lang.Class<java.lang.ref.ReferenceQueue>)
 
  at java.lang.Daemons$Daemon.run (Daemons.java:103)
 
  at java.lang.Thread.run (Thread.java:764)
"FinalizerWatchdogDaemon" tid=6 Sleeping
"FinalizerWatchdogDaemon" daemon prio=5 tid=6 Sleeping
  | group="system" sCount=1 dsCount=0 flags=1 obj=0x14a003c0 self=0xa3bdca00
  | sysTid=7798 nice=4 cgrp=default sched=0/0 handle=0x94cdf970
  | state=S schedstat=( 2072343 5941822 22 ) utm=0 stm=0 core=5 HZ=100
  | stack=0x94bdd000-0x94bdf000 stackSize=1038KB
  | held mutexes=
 
  at java.lang.Thread.sleep (Native method)
- sleeping on <0x0f998701> (a java.lang.Object)
 
  at java.lang.Thread.sleep (Thread.java:373)
- locked <0x0f998701> (a java.lang.Object)
 
  at java.lang.Thread.sleep (Thread.java:314)
 
  at java.lang.Daemons$FinalizerWatchdogDaemon.sleepFor (Daemons.java:342)
 
  at java.lang.Daemons$FinalizerWatchdogDaemon.waitForFinalization (Daemons.java:364)
 
  at java.lang.Daemons$FinalizerWatchdogDaemon.runInternal (Daemons.java:281)
 
  at java.lang.Daemons$Daemon.run (Daemons.java:103)
 
  at java.lang.Thread.run (Thread.java:764)
"Binder:7789_1" tid=8 Native
"Binder:7789_1" prio=5 tid=8 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14a00448 self=0xa2950c00
  | sysTid=7800 nice=0 cgrp=default sched=0/0 handle=0x949d7970
  | state=S schedstat=( 40129581 294155361 211 ) utm=2 stm=0 core=5 HZ=100
  | stack=0x948dd000-0x948df000 stackSize=1006KB
  | held mutexes=
 
  #00  pc 0000000000048c60  /system/lib/libc.so (__ioctl+8)
 
  #01  pc 000000000001dde5  /system/lib/libc.so (ioctl+32)
 
  #02  pc 0000000000042a4d  /system/lib/libbinder.so (android::IPCThreadState::talkWithDriver(bool)+168)
 
  #03  pc 0000000000042b45  /system/lib/libbinder.so (android::IPCThreadState::getAndExecuteCommand()+8)
 
  #04  pc 00000000000430c7  /system/lib/libbinder.so (android::IPCThreadState::joinThreadPool(bool)+38)
 
  #05  pc 0000000000058719  /system/lib/libbinder.so (???)
 
  #06  pc 000000000000d1c9  /system/lib/libutils.so (android::Thread::_threadLoop(void*)+144)
 
  #07  pc 000000000006e1f9  /system/lib/libandroid_runtime.so (android::AndroidRuntime::javaThreadShell(void*)+80)
 
  #08  pc 00000000000475af  /system/lib/libc.so (__pthread_start(void*)+22)
 
  #09  pc 000000000001afc9  /system/lib/libc.so (__start_thread+32)
"Binder:7789_2" tid=9 Native
"Binder:7789_2" prio=5 tid=9 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14a004d0 self=0xa2951200
  | sysTid=7801 nice=0 cgrp=default sched=0/0 handle=0x948da970
  | state=S schedstat=( 20506821 143575625 117 ) utm=1 stm=0 core=5 HZ=100
  | stack=0x947e0000-0x947e2000 stackSize=1006KB
  | held mutexes=
 
  #00  pc 0000000000048c60  /system/lib/libc.so (__ioctl+8)
 
  #01  pc 000000000001dde5  /system/lib/libc.so (ioctl+32)
 
  #02  pc 0000000000042a4d  /system/lib/libbinder.so (android::IPCThreadState::talkWithDriver(bool)+168)
 
  #03  pc 0000000000042b45  /system/lib/libbinder.so (android::IPCThreadState::getAndExecuteCommand()+8)
 
  #04  pc 00000000000430c7  /system/lib/libbinder.so (android::IPCThreadState::joinThreadPool(bool)+38)
 
  #05  pc 0000000000058719  /system/lib/libbinder.so (???)
 
  #06  pc 000000000000d1c9  /system/lib/libutils.so (android::Thread::_threadLoop(void*)+144)
 
  #07  pc 000000000006e1f9  /system/lib/libandroid_runtime.so (android::AndroidRuntime::javaThreadShell(void*)+80)
 
  #08  pc 00000000000475af  /system/lib/libc.so (__pthread_start(void*)+22)
 
  #09  pc 000000000001afc9  /system/lib/libc.so (__start_thread+32)
"Profile Saver" tid=10 Native
"Profile Saver" daemon prio=5 tid=10 Native
  | group="system" sCount=1 dsCount=0 flags=1 obj=0x14a00558 self=0xa2952400
  | sysTid=7802 nice=9 cgrp=default sched=0/0 handle=0x9408c970
  | state=S schedstat=( 62837913 237210212 145 ) utm=5 stm=0 core=6 HZ=100
  | stack=0x93f92000-0x93f94000 stackSize=1006KB
  | held mutexes=
 
  #00  pc 0000000000018d68  /system/lib/libc.so (syscall+28)
 
  #01  pc 00000000000b77d1  /system/lib/libart.so (art::ConditionVariable::WaitHoldingLocks(art::Thread*)+80)
 
  #02  pc 0000000000276d37  /system/lib/libart.so (art::ProfileSaver::Run()+342)
 
  #03  pc 00000000002790cf  /system/lib/libart.so (art::ProfileSaver::RunProfileSaverThread(void*)+50)
 
  #04  pc 00000000000475af  /system/lib/libc.so (__pthread_start(void*)+22)
 
  #05  pc 000000000001afc9  /system/lib/libc.so (__start_thread+32)
"UnityMain" tid=13 Native
"UnityMain" prio=5 tid=13 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14a005e0 self=0xa2c90200
  | sysTid=7807 nice=0 cgrp=default sched=0/0 handle=0x923eb970
  | state=S schedstat=( 33442360903 5191189563 31686 ) utm=2699 stm=643 core=5 HZ=100
  | stack=0x922e9000-0x922eb000 stackSize=1038KB
  | held mutexes=
 
  #00  pc 0000000000018d68  /system/lib/libc.so (syscall+28)
 
  #01  pc 00000000000b77d1  /system/lib/libart.so (art::ConditionVariable::WaitHoldingLocks(art::Thread*)+80)
 
  #02  pc 000000000027f1c5  /system/lib/libart.so (art::JNI::PushLocalFrame(_JNIEnv*, int)+328)
 
  #03  pc 00000000007b4310  /data/app/com.AccidentalEmpire.Hundredballs-AOtfHeBoPkaM7FvIn_juJQ==/lib/arm/libunity.so (???)
 
  at com.unity3d.player.UnityPlayer.nativeRender (Native method)
 
  at com.unity3d.player.UnityPlayer.c (unavailable)
 
  at com.unity3d.player.UnityPlayer$e$2.queueIdle (unavailable)
 
  at android.os.MessageQueue.next (MessageQueue.java:394)
 
  at android.os.Looper.loop (Looper.java:142)
 
  at com.unity3d.player.UnityPlayer$e.run (unavailable)
"queued-work-looper" tid=15 Native
"queued-work-looper" prio=5 tid=15 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14a006d8 self=0xa2c90e00
  | sysTid=7811 nice=-2 cgrp=default sched=0/0 handle=0x938db970
  | state=S schedstat=( 20996615 48859530 43 ) utm=0 stm=1 core=5 HZ=100
  | stack=0x937d9000-0x937db000 stackSize=1038KB
  | held mutexes=
 
  #00  pc 0000000000048b24  /system/lib/libc.so (__epoll_pwait+20)
 
  #01  pc 000000000001b3c5  /system/lib/libc.so (epoll_pwait+60)
 
  #02  pc 000000000001b3f5  /system/lib/libc.so (epoll_wait+12)
 
  #03  pc 000000000001002d  /system/lib/libutils.so (android::Looper::pollInner(int)+120)
 
  #04  pc 000000000000ff1d  /system/lib/libutils.so (android::Looper::pollOnce(int, int*, int*, void**)+32)
 
  #05  pc 00000000000b2795  /system/lib/libandroid_runtime.so (android::NativeMessageQueue::pollOnce(_JNIEnv*, _jobject*, int)+24)
 
  #06  pc 0000000000183ee5  /system/framework/arm/boot-framework.oat (Java_android_os_MessageQueue_nativePollOnce__JI+92)
 
  at android.os.MessageQueue.nativePollOnce (Native method)
 
  at android.os.MessageQueue.next (MessageQueue.java:325)
 
  at android.os.Looper.loop (Looper.java:142)
 
  at android.os.HandlerThread.run (HandlerThread.java:65)
"Binder:7789_3" tid=16 Native
"Binder:7789_3" prio=5 tid=16 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14a007c0 self=0x8e653400
  | sysTid=7842 nice=0 cgrp=default sched=0/0 handle=0x8c2ff970
  | state=S schedstat=( 23975987 120054220 163 ) utm=1 stm=0 core=6 HZ=100
  | stack=0x8c205000-0x8c207000 stackSize=1006KB
  | held mutexes=
 
  #00  pc 0000000000048c60  /system/lib/libc.so (__ioctl+8)
 
  #01  pc 000000000001dde5  /system/lib/libc.so (ioctl+32)
 
  #02  pc 0000000000042a4d  /system/lib/libbinder.so (android::IPCThreadState::talkWithDriver(bool)+168)
 
  #03  pc 0000000000042b45  /system/lib/libbinder.so (android::IPCThreadState::getAndExecuteCommand()+8)
 
  #04  pc 00000000000430c7  /system/lib/libbinder.so (android::IPCThreadState::joinThreadPool(bool)+38)
 
  #05  pc 0000000000058719  /system/lib/libbinder.so (???)
 
  #06  pc 000000000000d1c9  /system/lib/libutils.so (android::Thread::_threadLoop(void*)+144)
 
  #07  pc 000000000006e1f9  /system/lib/libandroid_runtime.so (android::AndroidRuntime::javaThreadShell(void*)+80)
 
  #08  pc 00000000000475af  /system/lib/libc.so (__pthread_start(void*)+22)
 
  #09  pc 000000000001afc9  /system/lib/libc.so (__start_thread+32)
"LooperProvider" tid=18 Native
"LooperProvider" prio=5 tid=18 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14a01a20 self=0x7b11c600
  | sysTid=8040 nice=0 cgrp=default sched=0/0 handle=0x7b0ff970
  | state=S schedstat=( 30687658 27328853 53 ) utm=2 stm=0 core=7 HZ=100
  | stack=0x7affd000-0x7afff000 stackSize=1038KB
  | held mutexes=
 
  #00  pc 0000000000048b24  /system/lib/libc.so (__epoll_pwait+20)
 
  #01  pc 000000000001b3c5  /system/lib/libc.so (epoll_pwait+60)
 
  #02  pc 000000000001b3f5  /system/lib/libc.so (epoll_wait+12)
 
  #03  pc 000000000001002d  /system/lib/libutils.so (android::Looper::pollInner(int)+120)
 
  #04  pc 000000000000ff1d  /system/lib/libutils.so (android::Looper::pollOnce(int, int*, int*, void**)+32)
 
  #05  pc 00000000000b2795  /system/lib/libandroid_runtime.so (android::NativeMessageQueue::pollOnce(_JNIEnv*, _jobject*, int)+24)
 
  #06  pc 0000000000183ee5  /system/framework/arm/boot-framework.oat (Java_android_os_MessageQueue_nativePollOnce__JI+92)
 
  at android.os.MessageQueue.nativePollOnce (Native method)
 
  at android.os.MessageQueue.next (MessageQueue.java:325)
 
  at android.os.Looper.loop (Looper.java:142)
 
  at android.os.HandlerThread.run (HandlerThread.java:65)
"GoogleApiHandler" tid=19 Native
"GoogleApiHandler" prio=5 tid=19 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14a016c8 self=0x8a4c7c00
  | sysTid=7925 nice=9 cgrp=default sched=0/0 handle=0x7b3ae970
  | state=S schedstat=( 43904279 259956977 143 ) utm=4 stm=0 core=5 HZ=100
  | stack=0x7b2ac000-0x7b2ae000 stackSize=1038KB
  | held mutexes=
 
  #00  pc 0000000000048b24  /system/lib/libc.so (__epoll_pwait+20)
 
  #01  pc 000000000001b3c5  /system/lib/libc.so (epoll_pwait+60)
 
  #02  pc 000000000001b3f5  /system/lib/libc.so (epoll_wait+12)
 
  #03  pc 000000000001002d  /system/lib/libutils.so (android::Looper::pollInner(int)+120)
 
  #04  pc 000000000000ff1d  /system/lib/libutils.so (android::Looper::pollOnce(int, int*, int*, void**)+32)
 
  #05  pc 00000000000b2795  /system/lib/libandroid_runtime.so (android::NativeMessageQueue::pollOnce(_JNIEnv*, _jobject*, int)+24)
 
  #06  pc 0000000000183ee5  /system/framework/arm/boot-framework.oat (Java_android_os_MessageQueue_nativePollOnce__JI+92)
 
  at android.os.MessageQueue.nativePollOnce (Native method)
 
  at android.os.MessageQueue.next (MessageQueue.java:325)
 
  at android.os.Looper.loop (Looper.java:142)
 
  at android.os.HandlerThread.run (HandlerThread.java:65)
"OkHttp ConnectionPool" tid=20 TimedWaiting
"OkHttp ConnectionPool" daemon prio=4 tid=20 TimedWaiting
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14a00848 self=0x8b508000
  | sysTid=7858 nice=10 cgrp=default sched=0/0 handle=0x89d5f970
  | state=S schedstat=( 30675206 354020049 109 ) utm=0 stm=2 core=7 HZ=100
  | stack=0x89c5d000-0x89c5f000 stackSize=1038KB
  | held mutexes=
 
  at java.lang.Object.wait (Native method)
- waiting on <0x0746f9a6> (a com.android.okhttp.ConnectionPool)
 
  at com.android.okhttp.ConnectionPool$1.run (ConnectionPool.java:101)
- locked <0x0746f9a6> (a com.android.okhttp.ConnectionPool)
 
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1162)
 
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:636)
 
  at java.lang.Thread.run (Thread.java:764)
"JavaBridge" tid=22 Native
"JavaBridge" prio=7 tid=22 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14a01938 self=0x8b508c00
  | sysTid=7945 nice=0 cgrp=default sched=0/0 handle=0x8b27f970
  | state=S schedstat=( 374380157 109683651 547 ) utm=29 stm=7 core=7 HZ=100
  | stack=0x8b17d000-0x8b17f000 stackSize=1038KB
  | held mutexes=
 
  #00  pc 0000000000048b24  /system/lib/libc.so (__epoll_pwait+20)
 
  #01  pc 000000000001b3c5  /system/lib/libc.so (epoll_pwait+60)
 
  #02  pc 000000000001b3f5  /system/lib/libc.so (epoll_wait+12)
 
  #03  pc 000000000001002d  /system/lib/libutils.so (android::Looper::pollInner(int)+120)
 
  #04  pc 000000000000ff1d  /system/lib/libutils.so (android::Looper::pollOnce(int, int*, int*, void**)+32)
 
  #05  pc 00000000000b2795  /system/lib/libandroid_runtime.so (android::NativeMessageQueue::pollOnce(_JNIEnv*, _jobject*, int)+24)
 
  #06  pc 0000000000183ee5  /system/framework/arm/boot-framework.oat (Java_android_os_MessageQueue_nativePollOnce__JI+92)
 
  at android.os.MessageQueue.nativePollOnce (Native method)
 
  at android.os.MessageQueue.next (MessageQueue.java:325)
 
  at android.os.Looper.loop (Looper.java:142)
 
  at android.os.HandlerThread.run (HandlerThread.java:65)
"UnityChoreographer" tid=23 Native
"UnityChoreographer" prio=5 tid=23 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14a00980 self=0x8b509200
  | sysTid=7859 nice=-2 cgrp=default sched=0/0 handle=0x89c5a970
  | state=S schedstat=( 2710430835 1584299489 5261 ) utm=178 stm=92 core=5 HZ=100
  | stack=0x89b58000-0x89b5a000 stackSize=1038KB
  | held mutexes=
 
  #00  pc 0000000000018d68  /system/lib/libc.so (syscall+28)
 
  #01  pc 00000000000b77d1  /system/lib/libart.so (art::ConditionVariable::WaitHoldingLocks(art::Thread*)+80)
 
  #02  pc 00000000003e1db9  /system/lib/libart.so (art::GoToRunnable(art::Thread*)+300)
 
  #03  pc 00000000003e1c5d  /system/lib/libart.so (art::JniMethodEnd(unsigned int, art::Thread*)+8)
 
  #04  pc 0000000000183eef  /system/framework/arm/boot-framework.oat (Java_android_os_MessageQueue_nativePollOnce__JI+102)
 
  at android.os.MessageQueue.nativePollOnce (Native method)
 
  at android.os.MessageQueue.next (MessageQueue.java:325)
 
  at android.os.Looper.loop (Looper.java:142)
 
  at android.os.HandlerThread.run (HandlerThread.java:65)
"Chrome_ProcessLauncherThread" tid=28 Native
"Chrome_ProcessLauncherThread" prio=5 tid=28 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14a00a68 self=0x8b57cc00
  | sysTid=7870 nice=0 cgrp=default sched=0/0 handle=0x85004970
  | state=S schedstat=( 16900577 11366768 86 ) utm=1 stm=0 core=7 HZ=100
  | stack=0x84f02000-0x84f04000 stackSize=1038KB
  | held mutexes=
 
  #00  pc 0000000000048b24  /system/lib/libc.so (__epoll_pwait+20)
 
  #01  pc 000000000001b3c5  /system/lib/libc.so (epoll_pwait+60)
 
  #02  pc 000000000001b3f5  /system/lib/libc.so (epoll_wait+12)
 
  #03  pc 000000000001002d  /system/lib/libutils.so (android::Looper::pollInner(int)+120)
 
  #04  pc 000000000000ff1d  /system/lib/libutils.so (android::Looper::pollOnce(int, int*, int*, void**)+32)
 
  #05  pc 00000000000b2795  /system/lib/libandroid_runtime.so (android::NativeMessageQueue::pollOnce(_JNIEnv*, _jobject*, int)+24)
 
  #06  pc 0000000000183ee5  /system/framework/arm/boot-framework.oat (Java_android_os_MessageQueue_nativePollOnce__JI+92)
 
  at android.os.MessageQueue.nativePollOnce (Native method)
 
  at android.os.MessageQueue.next (MessageQueue.java:325)
 
  at android.os.Looper.loop (Looper.java:142)
 
  at android.os.HandlerThread.run (HandlerThread.java:65)
"GoogleApiHandler" tid=29 Native
"GoogleApiHandler" prio=5 tid=29 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14a00b50 self=0x8b57d200
  | sysTid=7882 nice=9 cgrp=default sched=0/0 handle=0x84eff970
  | state=S schedstat=( 16262084 14187967 66 ) utm=0 stm=0 core=7 HZ=100
  | stack=0x84dfd000-0x84dff000 stackSize=1038KB
  | held mutexes=
 
  #00  pc 0000000000048b24  /system/lib/libc.so (__epoll_pwait+20)
 
  #01  pc 000000000001b3c5  /system/lib/libc.so (epoll_pwait+60)
 
  #02  pc 000000000001b3f5  /system/lib/libc.so (epoll_wait+12)
 
  #03  pc 000000000001002d  /system/lib/libutils.so (android::Looper::pollInner(int)+120)
 
  #04  pc 000000000000ff1d  /system/lib/libutils.so (android::Looper::pollOnce(int, int*, int*, void**)+32)
 
  #05  pc 00000000000b2795  /system/lib/libandroid_runtime.so (android::NativeMessageQueue::pollOnce(_JNIEnv*, _jobject*, int)+24)
 
  #06  pc 0000000000183ee5  /system/framework/arm/boot-framework.oat (Java_android_os_MessageQueue_nativePollOnce__JI+92)
 
  at android.os.MessageQueue.nativePollOnce (Native method)
 
  at android.os.MessageQueue.next (MessageQueue.java:325)
 
  at android.os.Looper.loop (Looper.java:142)
 
  at android.os.HandlerThread.run (HandlerThread.java:65)
"Chrome_IOThread" tid=30 Native
"Chrome_IOThread" prio=7 tid=30 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14a00c38 self=0x8b57ea00
  | sysTid=7887 nice=-4 cgrp=default sched=0/0 handle=0x84dfa970
  | state=S schedstat=( 1823437777 1182108561 4529 ) utm=115 stm=66 core=7 HZ=100
  | stack=0x84d00000-0x84d02000 stackSize=1006KB
  | held mutexes=
 
  #00  pc 0000000000048b24  /system/lib/libc.so (__epoll_pwait+20)
 
  #01  pc 000000000001b3c5  /system/lib/libc.so (epoll_pwait+60)
 
  #02  pc 000000000001b3f5  /system/lib/libc.so (epoll_wait+12)
 
  #03  pc 000000000003c5c9  /system/app/Chrome/Chrome.apk (???)
"GAC_Executor[0]" tid=32 Waiting
"GAC_Executor[0]" prio=5 tid=32 Waiting
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14a00d48 self=0x8b5f2800
  | sysTid=7901 nice=0 cgrp=default sched=0/0 handle=0x7bfb8970
  | state=S schedstat=( 5473328 32718702 33 ) utm=0 stm=0 core=7 HZ=100
  | stack=0x7beb6000-0x7beb8000 stackSize=1038KB
  | held mutexes=
 
  at java.lang.Object.wait (Native method)
- waiting on <0x05ad0fe7> (a java.lang.Object)
 
  at java.lang.Thread.parkFor$ (Thread.java:2137)
- locked <0x05ad0fe7> (a java.lang.Object)
 
  at sun.misc.Unsafe.park (Unsafe.java:358)
 
  at java.util.concurrent.locks.LockSupport.park (LockSupport.java:190)
 
  at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await (AbstractQueuedSynchronizer.java:2059)
 
  at java.util.concurrent.LinkedBlockingQueue.take (LinkedBlockingQueue.java:442)
 
  at java.util.concurrent.ThreadPoolExecutor.getTask (ThreadPoolExecutor.java:1087)
 
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1147)
 
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:636)
 
  at com.google.android.gms.internal.ef.run (unavailable)
 
  at java.lang.Thread.run (Thread.java:764)
"TaskSchedulerSi" tid=33 Native
"TaskSchedulerSi" prio=5 tid=33 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14a00f90 self=0x8b5f2e00
  | sysTid=7898 nice=0 cgrp=default sched=0/0 handle=0x7c2b7970
  | state=S schedstat=( 9994585 5703594 27 ) utm=0 stm=0 core=5 HZ=100
  | stack=0x7c1bd000-0x7c1bf000 stackSize=1006KB
  | held mutexes=
 
  #00  pc 0000000000018d68  /system/lib/libc.so (syscall+28)
 
  #01  pc 0000000000046f4b  /system/lib/libc.so (__pthread_cond_timedwait(pthread_cond_internal_t*, pthread_mutex_t*, bool, timespec const*)+102)
 
  #02  pc 0000000000039bb1  /system/app/Chrome/Chrome.apk (???)
"CleanupReference" tid=34 Waiting
"CleanupReference" daemon prio=5 tid=34 Waiting
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14a01018 self=0x8b5f3400
  | sysTid=7902 nice=0 cgrp=default sched=0/0 handle=0x7beb3970
  | state=S schedstat=( 334167 2844323 1 ) utm=0 stm=0 core=6 HZ=100
  | stack=0x7bdb1000-0x7bdb3000 stackSize=1038KB
  | held mutexes=
 
  at java.lang.Object.wait (Native method)
- waiting on <0x09325794> (a java.lang.Object)
 
  at java.lang.Object.wait (Object.java:422)
 
  at java.lang.ref.ReferenceQueue.remove (ReferenceQueue.java:188)
- locked <0x09325794> (a java.lang.Object)
 
  at java.lang.ref.ReferenceQueue.remove (ReferenceQueue.java:209)
 
  at org.chromium.android_webview.CleanupReference$1.run (CleanupReference.java:3)
"ConnectivityThread" tid=35 Native
"ConnectivityThread" prio=5 tid=35 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14a012d0 self=0x8b674600
  | sysTid=7903 nice=0 cgrp=default sched=0/0 handle=0x7bdae970
  | state=S schedstat=( 55081770 110778073 209 ) utm=3 stm=1 core=6 HZ=100
  | stack=0x7bcac000-0x7bcae000 stackSize=1038KB
  | held mutexes=
 
  #00  pc 0000000000048b24  /system/lib/libc.so (__epoll_pwait+20)
 
  #01  pc 000000000001b3c5  /system/lib/libc.so (epoll_pwait+60)
 
  #02  pc 000000000001b3f5  /system/lib/libc.so (epoll_wait+12)
 
  #03  pc 000000000001002d  /system/lib/libutils.so (android::Looper::pollInner(int)+120)
 
  #04  pc 000000000000ff1d  /system/lib/libutils.so (android::Looper::pollOnce(int, int*, int*, void**)+32)
 
  #05  pc 00000000000b2795  /system/lib/libandroid_runtime.so (android::NativeMessageQueue::pollOnce(_JNIEnv*, _jobject*, int)+24)
 
  #06  pc 0000000000183ee5  /system/framework/arm/boot-framework.oat (Java_android_os_MessageQueue_nativePollOnce__JI+92)
 
  at android.os.MessageQueue.nativePollOnce (Native method)
 
  at android.os.MessageQueue.next (MessageQueue.java:325)
 
  at android.os.Looper.loop (Looper.java:142)
 
  at android.os.HandlerThread.run (HandlerThread.java:65)
"AudioThread" tid=36 Native
"AudioThread" prio=5 tid=36 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14a013b8 self=0x8b675200
  | sysTid=7896 nice=0 cgrp=default sched=0/0 handle=0x7c4b1970
  | state=S schedstat=( 13422087 15983956 52 ) utm=0 stm=0 core=7 HZ=100
  | stack=0x7c3b7000-0x7c3b9000 stackSize=1006KB
  | held mutexes=
 
  #00  pc 0000000000018d6c  /system/lib/libc.so (syscall+32)
 
  #01  pc 0000000000046f4b  /system/lib/libc.so (__pthread_cond_timedwait(pthread_cond_internal_t*, pthread_mutex_t*, bool, timespec const*)+102)
 
  #02  pc 0000000000052455  /system/app/Chrome/Chrome.apk (???)
"Chrome_InProcGpuThread" tid=37 Native
"Chrome_InProcGpuThread" prio=7 tid=37 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14a01440 self=0x8b675800
  | sysTid=7905 nice=-4 cgrp=default sched=0/0 handle=0x7bca9970
  | state=S schedstat=( 1809504879 180027543 697 ) utm=165 stm=14 core=4 HZ=100
  | stack=0x7bbaf000-0x7bbb1000 stackSize=1006KB
  | held mutexes=
 
  #00  pc 0000000000018d6c  /system/lib/libc.so (syscall+32)
 
  #01  pc 0000000000046f4b  /system/lib/libc.so (__pthread_cond_timedwait(pthread_cond_internal_t*, pthread_mutex_t*, bool, timespec const*)+102)
 
  #02  pc 0000000000052455  /system/app/Chrome/Chrome.apk (???)
"TaskSchedulerBa" tid=39 Native
"TaskSchedulerBa" prio=5 tid=39 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14a01550 self=0x8b676400
  | sysTid=7891 nice=0 cgrp=default sched=0/0 handle=0x7c8a5970
  | state=S schedstat=( 8561877 5480052 41 ) utm=0 stm=0 core=6 HZ=100
  | stack=0x7c7ab000-0x7c7ad000 stackSize=1006KB
  | held mutexes=
 
  #00  pc 0000000000018d6c  /system/lib/libc.so (syscall+32)
 
  #01  pc 0000000000046f4b  /system/lib/libc.so (__pthread_cond_timedwait(pthread_cond_internal_t*, pthread_mutex_t*, bool, timespec const*)+102)
 
  #02  pc 0000000000052455  /system/app/Chrome/Chrome.apk (???)
"GAC_Executor[1]" tid=40 Waiting
"GAC_Executor[1]" prio=5 tid=40 Waiting
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14a015d8 self=0x8b674000
  | sysTid=7910 nice=0 cgrp=default sched=0/0 handle=0x7b7b8970
  | state=S schedstat=( 2088230 34876302 12 ) utm=0 stm=0 core=5 HZ=100
  | stack=0x7b6b6000-0x7b6b8000 stackSize=1038KB
  | held mutexes=
 
  at java.lang.Object.wait (Native method)
- waiting on <0x0196923d> (a java.lang.Object)
 
  at java.lang.Thread.parkFor$ (Thread.java:2137)
- locked <0x0196923d> (a java.lang.Object)
 
  at sun.misc.Unsafe.park (Unsafe.java:358)
 
  at java.util.concurrent.locks.LockSupport.park (LockSupport.java:190)
 
  at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await (AbstractQueuedSynchronizer.java:2059)
 
  at java.util.concurrent.LinkedBlockingQueue.take (LinkedBlockingQueue.java:442)
 
  at java.util.concurrent.ThreadPoolExecutor.getTask (ThreadPoolExecutor.java:1087)
 
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1147)
 
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:636)
 
  at com.google.android.gms.internal.ef.run (unavailable)
 
  at java.lang.Thread.run (Thread.java:764)
"pool-2-thread-1" tid=42 Waiting
"pool-2-thread-1" prio=5 tid=42 Waiting
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14a017b0 self=0x8a4c8800
  | sysTid=7926 nice=0 cgrp=default sched=0/0 handle=0x7b2a9970
  | state=S schedstat=( 33476197 78660210 80 ) utm=3 stm=0 core=5 HZ=100
  | stack=0x7b1a7000-0x7b1a9000 stackSize=1038KB
  | held mutexes=
 
  at java.lang.Object.wait (Native method)
- waiting on <0x05d90a32> (a java.lang.Object)
 
  at java.lang.Thread.parkFor$ (Thread.java:2137)
- locked <0x05d90a32> (a java.lang.Object)
 
  at sun.misc.Unsafe.park (Unsafe.java:358)
 
  at java.util.concurrent.locks.LockSupport.park (LockSupport.java:190)
 
  at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await (AbstractQueuedSynchronizer.java:2059)
 
  at java.util.concurrent.LinkedBlockingQueue.take (LinkedBlockingQueue.java:442)
 
  at java.util.concurrent.ThreadPoolExecutor.getTask (ThreadPoolExecutor.java:1087)
 
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1147)
 
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:636)
 
  at java.lang.Thread.run (Thread.java:764)
"AdWorker(NG) #1" tid=60 TimedWaiting
"AdWorker(NG) #1" prio=5 tid=60 TimedWaiting
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14a023b8 self=0x7b125e00
  | sysTid=8061 nice=0 cgrp=default sched=0/0 handle=0x79c9b970
  | state=# schedstat=( 7125729 20878959 43 ) utm=0 stm=0 core=-1 HZ=100
  | stack=0x79b99000-0x79b9b000 stackSize=1038KB
  | held mutexes=
 
  at java.lang.Object.wait (Native method)
- waiting on <0x0330c383> (a java.lang.Object)
 
  at java.lang.Thread.parkFor$ (Thread.java:2137)
- locked <0x0330c383> (a java.lang.Object)
 
  at sun.misc.Unsafe.park (Unsafe.java:358)
 
  at java.util.concurrent.locks.LockSupport.parkNanos (LockSupport.java:230)
 
  at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos (AbstractQueuedSynchronizer.java:2101)
 
  at java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take (ScheduledThreadPoolExecutor.java:1132)
 
  at java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take (ScheduledThreadPoolExecutor.java:1113)
 
  at java.util.concurrent.ThreadPoolExecutor.getTask (ThreadPoolExecutor.java:1087)
 
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1147)
 
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:636)
 
  at java.lang.Thread.run (Thread.java:764)
"pool-7-thread-1" tid=69 Waiting
"pool-7-thread-1" prio=5 tid=69 Waiting
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14a03118 self=0x8a55ea00
  | sysTid=8100 nice=10 cgrp=bg_non_interactive sched=0/0 handle=0x78b66970
  | state=S schedstat=( 643597490 807381470 2616 ) utm=39 stm=24 core=4 HZ=100
  | stack=0x78a64000-0x78a66000 stackSize=1038KB
  | held mutexes=
 
  at java.lang.Object.wait (Native method)
- waiting on <0x0c92cbf5> (a java.lang.Object)
 
  at java.lang.Thread.parkFor$ (Thread.java:2137)
- locked <0x0c92cbf5> (a java.lang.Object)
 
  at sun.misc.Unsafe.park (Unsafe.java:358)
 
  at java.util.concurrent.locks.LockSupport.park (LockSupport.java:190)
 
  at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await (AbstractQueuedSynchronizer.java:2059)
 
  at java.util.concurrent.LinkedBlockingQueue.take (LinkedBlockingQueue.java:442)
 
  at java.util.concurrent.ThreadPoolExecutor.getTask (ThreadPoolExecutor.java:1087)
 
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1147)
 
  at java.util.concurrent.ThreadPoolExecutor$Worker.run
"Thread-30" tid=70 Waiting
"Thread-30" prio=5 tid=70 Waiting
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14a028d0 self=0x7932c600
  | sysTid=8093 nice=10 cgrp=default sched=0/0 handle=0x7907f970
  | state=S schedstat=( 23123286 30735778 36 ) utm=1 stm=0 core=4 HZ=100
  | stack=0x78f7d000-0x78f7f000 stackSize=1038KB
  | held mutexes=
 
  at java.lang.Object.wait (Native method)
- waiting on <0x01a75900> (a java.lang.Object)
 
  at java.lang.Thread.parkFor$ (Thread.java:2137)
- locked <0x01a75900> (a java.lang.Object)
 
  at sun.misc.Unsafe.park (Unsafe.java:358)
 
  at java.util.concurrent.locks.LockSupport.park (LockSupport.java:190)
 
  at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await (AbstractQueuedSynchronizer.java:2059)
 
  at java.util.concurrent.PriorityBlockingQueue.take (PriorityBlockingQueue.java:548)
 
  at ks.b (com.google.android.gms.policy_ads_fdr_dynamite@20290004@20290004.249178941.249178941:1)
 
  at ks.run (com.google.android.gms.policy_ads_fdr_dynamite@20290004@20290004.249178941.249178941:3)
"Thread-33" tid=71 Waiting
"Thread-33" prio=5 tid=71 Waiting
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14a02c90 self=0x7932d800
  | sysTid=8096 nice=10 cgrp=default sched=0/0 handle=0x78d70970
  | state=S schedstat=( 445364 813282 3 ) utm=0 stm=0 core=6 HZ=100
  | stack=0x78c6e000-0x78c70000 stackSize=1038KB
  | held mutexes=
 
  at java.lang.Object.wait (Native method)
- waiting on <0x075f1539> (a java.lang.Object)
 
  at java.lang.Thread.parkFor$ (Thread.java:2137)
- locked <0x075f1539> (a java.lang.Object)
 
  at sun.misc.Unsafe.park (Unsafe.java:358)
 
  at java.util.concurrent.locks.LockSupport.park (LockSupport.java:190)
 
  at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await (AbstractQueuedSynchronizer.java:2059)
 
  at java.util.concurrent.PriorityBlockingQueue.take (PriorityBlockingQueue.java:548)
 
  at lc.a (com.google.android.gms.policy_ads_fdr_dynamite@20290004@20290004.249178941.249178941:1)
 
  at lc.run (com.google.android.gms.policy_ads_fdr_dynamite@20290004@20290004.249178941.249178941:2)
"Thread-31" tid=72 Waiting
"Thread-31" prio=5 tid=72 Waiting
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14a02ed8 self=0x7932cc00
  | sysTid=8094 nice=10 cgrp=default sched=0/0 handle=0x78f7a970
  | state=S schedstat=( 368072 2562605 5 ) utm=0 stm=0 core=6 HZ=100
  | stack=0x78e78000-0x78e7a000 stackSize=1038KB
  | held mutexes=
 
  at java.lang.Object.wait (Native method)
- waiting on <0x0f09977e> (a java.lang.Object)
 
  at java.lang.Thread.parkFor$ (Thread.java:2137)
- locked <0x0f09977e> (a java.lang.Object)
 
  at sun.misc.Unsafe.park (Unsafe.java:358)
 
  at java.util.concurrent.locks.LockSupport.park (LockSupport.java:190)
 
  at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await (AbstractQueuedSynchronizer.java:2059)
 
  at java.util.concurrent.PriorityBlockingQueue.take (PriorityBlockingQueue.java:548)
 
  at lc.a (com.google.android.gms.policy_ads_fdr_dynamite@20290004@20290004.249178941.249178941:1)
 
  at lc.run (com.google.android.gms.policy_ads_fdr_dynamite@20290004@20290004.249178941.249178941:2)
"Thread-32" tid=73 Waiting
"Thread-32" prio=5 tid=73 Waiting
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14a02f98 self=0x7932d200
  | sysTid=8095 nice=10 cgrp=default sched=0/0 handle=0x78e75970
  | state=S schedstat=( 306511 2015156 4 ) utm=0 stm=0 core=6 HZ=100
  | stack=0x78d73000-0x78d75000 stackSize=1038KB
  | held mutexes=
 
  at java.lang.Object.wait (Native method)
- waiting on <0x0cb214df> (a java.lang.Object)
 
  at java.lang.Thread.parkFor$ (Thread.java:2137)
- locked <0x0cb214df> (a java.lang.Object)
 
  at sun.misc.Unsafe.park (Unsafe.java:358)
 
  at java.util.concurrent.locks.LockSupport.park (LockSupport.java:190)
 
  at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await (AbstractQueuedSynchronizer.java:2059)
 
  at java.util.concurrent.PriorityBlockingQueue.take (PriorityBlockingQueue.java:548)
 
  at lc.a (com.google.android.gms.policy_ads_fdr_dynamite@20290004@20290004.249178941.249178941:1)
 
  at lc.run (com.google.android.gms.policy_ads_fdr_dynamite@20290004@20290004.249178941.249178941:2)
"Thread-34" tid=74 Waiting
"Thread-34" prio=5 tid=74 Waiting
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x14a03058 self=0x7932de00
  | sysTid=8097 nice=10 cgrp=default sched=0/0 handle=0x78c6b970
  | state=S schedstat=( 387656 1320000 4 ) utm=0 stm=0 core=6 HZ=100
  | stack=0x78b69000-0x78b6b000 stackSize=1038KB
  | held mutexes=
 
  at java.lang.Object.wait (Native method)
- waiting on <0x0da9052c> (a java.lang.Object)
 
  at java.lang.Thread.parkFor$ (Thread.java:2137)
- locked <0x0da9052c> (a java.lang.Object)
 
  at sun.misc.Unsafe.park (Unsafe.java:358)
 
  at java.util.concurrent.locks.LockSupport.park (LockSupport.java:190)
 
  at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await (AbstractQueuedSynchronizer.java:2059)
 
  at java.util.concurrent.PriorityBlockingQueue.take (PriorityBlockingQueue.java:548)
 
  at lc.a (com.google.android.gms.policy_ads_fdr_dynamite@20290004@20290004.249178941.249178941:1)
 
  at lc.run (com.google.android.gms.policy_ads_fdr_dynamite@20290004@20290004.249178941.249178941:2)
"Signal Catcher" tid=3 Runnable
"Signal Catcher" daemon prio=5 tid=3 Runnable
  | group="system" sCount=0 dsCount=0 flags=0 obj=0x14a00228 self=0xac836200
  | sysTid=7795 nice=0 cgrp=default sched=0/0 handle=0xa5396970
  | state=R schedstat=( 132644579 7327970 93 ) utm=4 stm=8 core=7 HZ=100
  | stack=0xa529c000-0xa529e000 stackSize=1006KB
  | held mutexes= "mutator lock"(shared held)
 
  #00  pc 00000000002e9d9f  /system/lib/libart.so (art::DumpNativeStack(std::__1::basic_ostream<char, std::__1::char_traits<char>>&, int, BacktraceMap*, char const*, art::ArtMethod*, void*)+130)
 
  #01  pc 000000000037abe1  /system/lib/libart.so (art::Thread::DumpStack(std::__1::basic_ostream<char, std::__1::char_traits<char>>&, bool, BacktraceMap*, bool) const+204)
 
  #02  pc 0000000000377317  /system/lib/libart.so (art::Thread::Dump(std::__1::basic_ostream<char, std::__1::char_traits<char>>&, bool, BacktraceMap*, bool) const+34)
 
  #03  pc 000000000038ecab  /system/lib/libart.so (art::DumpCheckpoint::Run(art::Thread*)+698)
 
  #04  pc 0000000000388825  /system/lib/libart.so (art::ThreadList::RunCheckpoint(art::Closure*, art::Closure*)+320)
 
  #05  pc 0000000000388323  /system/lib/libart.so (art::ThreadList::Dump(std::__1::basic_ostream<char, std::__1::char_traits<char>>&, bool)+530)
 
  #06  pc 0000000000388043  /system/lib/libart.so (art::ThreadList::DumpForSigQuit(std::__1::basic_ostream<char, std::__1::char_traits<char>>&)+626)
 
  #07  pc 0000000000365f23  /system/lib/libart.so (art::Runtime::DumpForSigQuit(std::__1::basic_ostream<char, std::__1::char_traits<char>>&)+122)
 
  #08  pc 000000000036d01f  /system/lib/libart.so (art::SignalCatcher::HandleSigQuit()+1282)
 
  #09  pc 000000000036bffd  /system/lib/libart.so (art::SignalCatcher::Run(void*)+240)
 
  #10  pc 00000000000475af  /system/lib/libc.so (__pthread_start(void*)+22)
 
  #11  pc 000000000001afc9  /system/lib/libc.so (__start_thread+32)
"HeapTaskDaemon" tid=7 Blocked
"HeapTaskDaemon" daemon prio=5 tid=7 Blocked
  | group="system" sCount=1 dsCount=0 flags=1 obj=0x14a04a50 self=0xa2940000
  | sysTid=7799 nice=4 cgrp=default sched=0/0 handle=0x94bda970
  | state=S schedstat=( 814345373 666328800 1046 ) utm=69 stm=11 core=7 HZ=100
  | stack=0x94ad8000-0x94ada000 stackSize=1038KB
  | held mutexes=
 
  #00  pc 0000000000018d68  /system/lib/libc.so (syscall+28)
 
  #01  pc 00000000000b77d1  /system/lib/libart.so (art::ConditionVariable::WaitHoldingLocks(art::Thread*)+80)
 
  #02  pc 00000000001cb29f  /system/lib/libart.so (art::gc::TaskProcessor::GetTask(art::Thread*)+278)
 
  #03  pc 00000000001cb869  /system/lib/libart.so (art::gc::TaskProcessor::RunAllTasks(art::Thread*)+44)
 
  #04  pc 0000000000193e2b  /system/framework/arm/boot-core-libart.oat (Java_dalvik_system_VMRuntime_runHeapTasks__+74)
 
  at dalvik.system.VMRuntime.runHeapTasks (Native method)
- waiting to lock an unknown object
 
  at java.lang.Daemons$HeapTaskDaemon.runInternal (Daemons.java:461)
 
  at java.lang.Daemons$Daemon.run (Daemons.java:103)
 
  at java.lang.Thread.run (Thread.java:764)
```

**Comment by olehkuznetsov** [comment]
Not sure if it is related to Play Games plugin.

**Comment by vaibhavgpt-google** [comment]
Closing this issue as part of a cleanup of older tickets. Google Play Games v1 is now deprecated. Please upgrade to version v2.1.0 or newer: https://github.com/playgameservices/play-games-plugin-for-unity/releases/tag/v2.1.0. If this issue persists on the new v2 architecture, please open a new ticket.
